### PR TITLE
Bug 1163784 - Hightlight Bluetooth icon while profile connection changed

### PR DESCRIPTION
**Pull request by ian-liu**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=926e5eab71c38af0d3f239c284c89bbe8a35f5e8

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=05535ea6692c94ca07c7c0bd90677e8b2cacf0d9

File: apps/system/js/bluetooth_v2.js [hunk 1]
```diff
@@ -103,6 +103,8 @@ Bluetooth.prototype = {
           }
         }
       ));
+      // Update Bluetooth and profile icons
+      this.icon && this.icon.update();
```
Comment (gasolin) [inline apps/system/js/bluetooth_v2.js]: lgtm :+1: , can you add a line of test to make sure its called?

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bddf92b68eaea8a37f23115d2d6ccd3e92a46c76

File: apps/system/js/bluetooth_v2.js (hunk 1 above)
Comment (ian-liu) [inline apps/system/js/bluetooth_v2.js]: Done. Thanks for reminder here!

**Comment by ian-liu** [comment]
Passed on try-server. We can merge the pull request now.

**Pull request merged**